web/router.ex

```diff
@@ -63,6 +63,7 @@ defmodule CodeCorps.Router do
     resources "/categories", CategoryController, only: [:create, :update]
     resources "/comments", CommentController, only: [:create, :update]
     resources "/donation-goals", DonationGoalController, only: [:create, :update, :delete]
+    post "/github_connect", UserController, :github_connect
```

Comment:
This is on me, because I'm the one that wrote it in the issue but could you change the path to `/github-connect`,  so it's consistent with other parts of the app. I keep mixing up my dashes and underscores.